framework/SupervisedLearning/DynamicModeDecompositionControl.py

```diff
@@ -384,7 +400,18 @@ def _evaluateMatrices(self, X1, X2, U, Y1, rankSVD):
     # Omega Matrix, stack X1 and U
     omega = np.concatenate((X1, U), axis=0)
     # SVD
-    Ut, St, Vt = mathUtils.computeTruncatedSingularValueDecomposition(omega, rankSVD, True, False)
+    Utsvd, stsvd, Vtsvd = mathUtils.computeTruncatedSingularValueDecomposition(omega, rankSVD, False, False)
+    # Find the truncation rank triggered by "s>=SminValue"
+    rank_s = sum(map(lambda x : x>=1e-6, stsvd.tolist()))
```

Comment:
underscore is not allowed in variables, please use rankTruc instead of rank_s here

Reply:
Done